app/src/androidTest/java/com/github/onlynotesswent/ui/overview/OverviewTest.kt

```diff
@@ -248,6 +249,37 @@ class OverviewTest {
     verify(navigationActions).navigateTo(screen = Screen.EDIT_NOTE)
   }
 
+  @Test
+  fun openFileSystem() {
+    composeTestRule.onNodeWithTag("FileSystemPopup").assertIsNotDisplayed()
+    composeTestRule.onNodeWithTag("showFileSystemButton").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("showFileSystemButton").performClick()
+    composeTestRule.onNodeWithTag("FileSystemPopup").assertIsDisplayed()
+  }
+
+  @Test
+  fun navigateFileSystem() {
+    val userRootFoldersFlow =
+        listOf(
+            Folder(id = "8", name = "Root Folder 1", userId = "1", parentFolderId = "1"),
+            Folder(id = "9", name = "Root Folder 2", userId = "2", parentFolderId = "1"))
```

Comment:
For the second folder, the userId should be "1" to have it belong to user "1".

Reply:
Yes I will change that, I wrote random user numbers as I do not really care about them in my tests but it is more coherent to have the same user number.